charts/pxc-operator/Chart.yaml

```diff
@@ -1,9 +1,9 @@
-apiVersion: v1
-appVersion: "1.8.0"
+apiVersion: v2
+appVersion: "1.9.0"
```

Comment:
I see in pxc-db we have changed `appVersion` to be without quotes, maybe just make it consistent.

Reply:
done